local/rest_api_gcbm/Dockerfile.cml

```diff
@@ -0,0 +1,24 @@
+FROM ghcr.io/moja-global/flint.gcbm:master
```

Comment:
I think we can merge this into the `rest_api_gcbm` Dockerfile (above). Then we wouldn't need to rebuild it as part of the Action.